text/0000-lts.md

```diff
@@ -0,0 +1,164 @@
+bug justifying a patch exists in any of these channels, we will release a patch
+for that channel, just as we do today for stable. However, lts releases are
+less likely to receive patch releases in practice, since these bugs are most
+often found in the first six weeks after a stable release.
+
+As a result of this, a stable release which is a candidate for the lts channel
+(that is, its minor version is a multiple of four) will be supported across all
+three channels for the following nine release cycles, a period of 54 weeks or
+just over a year.
+
+## Recommendations for library authors
+
+Today, library authors fall into three general camps:
+
+1. Their library compiles on the nightly channel.
+2. Their library compiles on the stable channel.
+3. Their library compiles on any release more recent than an arbitrary old
+release, which is different for each library (for example, `failure` compiles
+on 1.18.0, whereas `serde` compiles on 1.13.0).
+
+Frequently, whichever level of support they offer, they guarantee it by testing
+their code on that channel or release using CI.
+
+The Rust project recommends that library authors interested in providing
+support for users on versions older than the latest stable, instead of focusing
+on supporting particular *versions*, focus on supporting particular *release
+channels*. That is, a library may guarantee that it will compile on any of the
+stable release channels, determining how old the version of Rust their users
+can be on with a guarantee of compatibility.
+
+We recommend that libraries consider dropping support for a release channel to
+be a breaking change, but that dropping support for any particular version of
+Rust to be a non-breaking change. 
+
+If a library does not support a particular release channel, effectively all of
+the libraries that depend on *it* also do not support that channel. For this
+reason, we strongly recommend that so-called "core libraries" - those on which
+many other libraries depend - adopt a practice of supporting all three stable
+release channels. Examples of core libraries include libc and serde.
+
+For libraries that do not believe they are "core libraries," we make no
+particular recommendation about which channels to support. There is always a
+trade off between providing the best API for users on stable as soon as
+possible vs supporting users who choose to stay on older versions of Rust, and
+each library should make a judgment call regarding the trade off in their
+particular circumstances.
+
+# Reference-level explanation
+[reference-level-explanation]: #reference-level-explanation
+
+Current policy is that the core team decides if a bugfix justifies a patch
+release to the stable channel. This policy will remain the same for patches to
+the lts and lts-prior channels.
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+This increases the maintenance burden on the release and infrastructure teams
+by adding two additional release channels.
```

Comment:
I'm currently skeptical that the bandwidth to do this release work and that the funding to support those doing the work exists. But I would be happy to be proven wrong.